### PR TITLE
1641.release rc4

### PR DESCRIPTION
**Pull request by daira**
Release 1.0.0-rc4. Updates pchMessageStart strings for mainnet and testnet.

### CODE REVIEW

File: doc/release-notes/release-notes-1.0.0-rc4.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Daira Hopwood (3):
+      Update pchMessageStart for mainnet and testnet.
+      Update version numbers for 1.0.0-rc4.
+      Add release notes for 1.0.0-rc4.
+
+Jack Grigg (4):
```
Comment (bitcartel) [inline doc/release-notes/release-notes-1.0.0-rc4.md]: None of the below is currently present in the commits.

Comment (bitcartel) [inline doc/release-notes/release-notes-1.0.0-rc4.md]: I don't see the below in the commits.

File: doc/release-notes/release-notes-1.0.0-rc4.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      Integrate production Founders' Reward keys
+      Remove Founders' Reward override from #1398
+      Regenerate mainnet and testnet genesis blocks for nMaxTipAge change
+      Update tests for new genesis blocks
+
+Sean Bowe (1):
+      Zcash zk-SNARK public parameters for 1.0 "Sprout".
```
Comment (bitcartel) [inline doc/release-notes/release-notes-1.0.0-rc4.md]: Don't see this in commits.

File: doc/release-notes/release-notes-1.0.0-rc4.md (hunk 1 above)
Comment (daira) [inline doc/release-notes/release-notes-1.0.0-rc4.md]: https://github.com/zcash/zcash/commit/ab3225490976c017bd09f747feeedddf3acb701e

File: doc/release-notes/release-notes-1.0.0-rc4.md (hunk 2 above)
Comment (daira) [inline doc/release-notes/release-notes-1.0.0-rc4.md]: https://github.com/zcash/zcash/commit/278439fbe3470f36eb8bff538bdadddb1bf6b0e8

**Comment by bitcartel** [comment]
ACK

File: doc/release-notes/release-notes-1.0.0-rc4.md (hunk 2 above)
Comment (nathan-at-least) [inline doc/release-notes/release-notes-1.0.0-rc4.md]: I don't understand this comment, @daira.

Comment (daira) [inline doc/release-notes/release-notes-1.0.0-rc4.md]: @bitcartel had made a comment that he didn't see the relevant commit; I think he deleted that comment (at least I don't see it), so my response was confusing out of context.

**Comment by daira** [comment]
@zkbot r+

File: doc/release-notes/release-notes-1.0.0-rc4.md (hunk 2 above)
Comment (nathan-at-least) [inline doc/release-notes/release-notes-1.0.0-rc4.md]: Ok, then I amend my review to a full ACK-without-testing.

**Pull request merged**